helm/flink-kubernetes-operator/conf/log4j-console.properties

```diff
@@ -60,3 +60,6 @@ appender.rolling.strategy.max = 10
 # Suppress the irrelevant (wrong) warnings from the Netty channel handler
 logger.netty.name = org.apache.flink.shaded.akka.org.jboss.netty.channel.DefaultChannelPipeline
 logger.netty.level = OFF
+
+# Uncomment this if you want to allow dynamically change logging level
+# monitorInterval = 30
```

Comment:
Question: Should we enable it by default?
Same question to `log4j-operator.properties`.

Reply:
Yes probably as thats what people expect of configMaps for them to be editable and picked up without recreating the pod.